SMFloatingLabelTextField/Classes/SMFloatingLabelTextField.m

```diff
@@ -52,8 +52,9 @@ - (void)setup {
 - (void)setupInitialInspectableAttributesValues {
     self.floatingLabelPassiveColor = [UIColor lightGrayColor];
     self.floatingLabelActiveColor = [UIColor blueColor];
-    self.floatingLabelFont = [UIFont systemFontOfSize:12.0];
+    self.floatingLabelFont = [UIFont preferredFontForTextStyle:UIFontTextStyleFootnote];
```

Comment:
Default text field text style is Body and Footnote match closely the size we need from floating labels font related to the text fields font.